test/e2e/common.go

```diff
@@ -57,6 +62,17 @@ func setupSpecNamespace(ctx context.Context, specName string, clusterProxy frame
 	return namespace, cancelWatches
 }
 
+func setCNIResources(cniManifestPath string) {
```

Comment:
This method is always called with CNI path as input and it is called for every test.
Wondering if we can make this a little bit more transparent and
- make `setCNIResources` automatically happen during E2Econfig load
- if CNIPath exists, automatically load it into CNI_RESOURCES (or CNI_YAML)